jbpm-db-scripts/src/main/resources/db/upgrade-scripts/postgresql/jbpm-7.58-to-7.59.sql

```diff
@@ -0,0 +1,5 @@
+create unique index IDX_CorrelationKeyInfo_name on CorrelationKeyInfo(name);
```

Comment:
Same question for postgresql.

Reply:
I was suprised by that, but only happens with the autogenerated means (bytea and new spring boot scripts). We cannot do much about it but add some release notes